src/styles/index.ts

```diff
@@ -4166,10 +4174,13 @@ const styles = (theme: ThemeColors) =>
             display: 'flex',
             justifyContent: 'center',
             alignItems: 'center',
-            backgroundColor: theme.border,
```

Comment:
We could've easily over-written with `spread` operator. Instead of deleting this, as this could affect styles in other places where its used.